rules/windows/persistence_suspicious_service_created_registry.toml

```
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2020/11/23"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of a suspicious ImagePath value. This could be an indication of an adversary attempting to
+stealthily persist or elevate privileges through abnormal service creation.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Suspicious Service ImagePath Created"
+risk_score = 71
+rule_id = "36a8e048-d888-4f61-a8b9-0f9e2e40f317"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+registry where wildcard(registry.path, "HKLM\\SYSTEM\\ControlSet*\\Services\\*\\ImagePath") and
+ /* add suspicious registry ImagePath values here */
+ wildcard(registry.data.strings, "%COMSPEC%*", "*\\.\\pipe\\*")
+'''
+
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1543.003"
+name = "Windows Service"
```

Not sure if this can be changed or if it hurts Unit Tests?
```suggestion
name = "Create or Modify System Process: Windows Service"
```